client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx

```diff
@@ -2,30 +2,41 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import Gridicon from 'gridicons';
 import { localize } from 'i18n-calypso';
+import ThankYouCard from 'components/thank-you-card';
 
 class RebrandCitiesThankYou extends Component {
+	renderLogo() {
+		return <img src="/calypso/images/rebrand-cities/wp-rebrand-cities-logo.svg" />;
```

Comment:
The SVG was way too big to add here so I included it as an image. This cases the image to "flash" into view after it loads. Any thoughts of another way to do this?